src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -78,6 +80,35 @@
           </div>
         </div>
         <br class="hidden-xs">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="form-group">
+              <h5 class="col-sm-2">
+                <label class="control-label" for="<%= Const.ParamsNames.COURSE_NAME %>">
+                  Course name
+                </label>
+              </h5>
+              <div class="col-sm-10 col-md-8">
+                <c:choose>
+                  <c:when test="${fsForm.courseIdEditable}">
+                    <c:forEach items="${courseAttributes}" var="attributes">
+                      <input class="course-name-data" type="hidden"
+                             value="[${attributes.id}][${attributes.name}]">
```

Comment:
The indentation rule is 4 spaces for line breaks, not aligning with the attributes. Also, this looks short enough to fit the previous line.